sr_robot_launch/scripts/construct_robot_description

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2020 Shadow Robot Company Ltd - All Rights Reserved. Proprietary and Confidential.
+# Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+
+import os
+import rospy
+import rospkg
+from subprocess import check_output
+from sr_ur_arm_calibration_loader.sr_ur_arm_calibration_loader import SrUrLoadCalibration
+
+
+class SrConstructRobotDescription():
+    def __init__(self):
+        self._robot_description_file = rospy.get_param("~robot_description_file")
+        arm_type = rospy.get_param("~arm_type")
+        self._bimanual = False
+        self._xacro_commands = []
+        self._node_name = rospy.get_name()
+
+        if 'bimanual' in self._robot_description_file:
+            self._bimanual = True
+
+        arm1 = ('ra', '192.168.1.1', arm_type)
```

Comment:
for constants please follow convention CONST_NAME_OF VARIABLE
also, think its clearer to name ra_arm and la_arm. also, please use dictionary instead of tuple for more clarity